src/Chainweb/Mempool/InMem.hs

```diff
@@ -96,9 +97,10 @@ newInMemMempoolData =
 
 ------------------------------------------------------------------------------
 toMempoolBackend
-    :: InMemoryMempool t
+    :: ChainwebVersion
+    -> InMemoryMempool t
     -> IO (MempoolBackend t)
-toMempoolBackend mempool = do
+toMempoolBackend v mempool = do
```

Comment:
I wonder if it would be better to just add the chainweb version to `InMemConfig` and make it an instance of `HasChainwebVersion`.